src/exodus/tx.cpp

```diff
@@ -2335,6 +2361,59 @@ int CMPTransaction::logicMath_UnfreezeTokens()
     return 0;
 }
 
+/** Tx 1025 */
+int CMPTransaction::logicMath_CreateDenomination()
+{
+    if (!IsTransactionTypeAllowed(block, property, type, version)) {
+        PrintToLog("%s(): rejected: type %d or version %d not permitted for property %d at block %d\n",
+                __func__,
+                type,
+                version,
+                property,
+                block);
+        return PKT_ERROR_TOKENS - 22;
+    }
+
+    if (!nValue || nValue > MAX_INT_8_BYTES) {
```

Comment:
if change nValue to int64_t this should be changed to `< 0`

Reply:
We can't do that for now due to it need to check a lot. Will be better to refactor it later.